src/dashboards.js

```diff
@@ -18,6 +18,36 @@ function setConfig(key, value, callback) {
     fs.writeFile(CONFIG_PATH, JSON.stringify(config), callback);
 }
 
+async function getOSDStatus() {
+    let osStatus = await curlCommand('curl localhost:9200');
```

Comment:
ok for POC, but what if OS doesn't has `curl` e.g. windows. 
let's use actual request library like `https://www.npmjs.com/package/node-fetch`
```
import fetch from 'node-fetch';

const response = await fetch('https://github.com/');
const body = await response.text();

console.log(body);
```